src/parse.rs

```diff
@@ -277,6 +278,12 @@ enum TableParseMode {
     Disabled,
 }
 
+pub struct BrokenLink<'a> {
+    pub span: ::std::ops::Range<usize>,
+    pub link_type: LinkType,
+    pub reference: &'a CowStr<'a>,
```

Comment:
This seems really strange - it's a reference to a reference. I think `&'a str` would do as well, and that can point to the contents of a CowStr. But maybe I'm missing something here.

Reply:
You're not missing anything. In fact, you've caught something ;-) Thanks.